src/app/pages/instances/components/all-instances/all-instances-header/global-config-form/global-config-form.component.html

```diff
@@ -14,6 +14,13 @@
           [required]="true"
           [options]="poolOptions$"
         ></ix-select>
+
+        <ix-select
```

Comment:
I think this needs to be reworked.
I would go with:
```
[ ] Enabled checkbox
Pools multiselect
```

Pools multiselect should be hidden when feature is not enabled. There is no value in showing default pool, as we still going to show the same field in all other forms.